airflow/dag_processing/processor.py

```diff
@@ -187,6 +188,18 @@ def _handle_dag_file_processing():
 
     def start(self) -> None:
         """Launch the process and start processing the DAG."""
+        import importlib
```

Comment:
Why is this import local, not top-level?

Reply:
hmm, no specific reason, it's really necessary for this specific part of the code, shouldn't be relevant for any other function in this file, but I can certainly make it top level if you think it's better.

Reply:
Generally we prefer keeping things imported at the top unless for performance reasons. `importlib` is very lightweight.

Reply:
ok gotcha, I'll move it.